tests_integ/single/test_basic_api.py

```diff
@@ -0,0 +1,18 @@
+import os
+
```

Comment:
Generally - why so many different classes? These all appear to do basically the same thing, why not one class with a long parameterized list unless there are material differences?

Reply:
For this initial version, we just test all the templates with `create_and_verify_stack` function. We will add specific steps for tests later and that's why we group them into different classes.